_posts/2018-05-21-event-store-4.1.1-hotfix-1-release.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Event Store 4.1.1 Hotfix 1"
+author: "Shaan Nobee"
+layout: blog-post
+---
+
+A hotfix for Event Store 4.1.1 has been released to fix the following issue:
+https://github.com/EventStore/EventStore/issues/1627
+
+The hotfix can be downloaded for :
+- Windows (via Chocolatey)
+- Ubuntu 18.04 (via packagecloud)
+- Ubuntu 16.04 (via packagecloud)
+- Ubuntu 14.04 (via packagecloud)
+
+**Issue:**  
+Due to a mis-configuration, logging is being done synchronously on version 4.1.1  
```

Comment:
@shaan1337 "misconfiguration", and again this is passive voice, but slightly harder to fix, try:

> …, version 4.1.1 logs synchronously